Git process.md

```diff
@@ -29,3 +29,69 @@ While the application is still not deployed on a production server, you can omit
 **Make small commits, following the Single Responsibility Principle in Git.** This makes commits easier to review when making pull requests, and it's easier to notice what's going on when something goes wrong.
 
 **Don't use `git add .`** Review what you're adding to your repo—this is the #1 cause of making unwanted changes.
+
+### Github Pull Request Descriptions
+
+It's essential to write a good Pull Request description. Reviewers will usually read it before
+looking at the diff, so make sure it gives them enough context, so they know what they are looking at.
+
+Another purpose of a good description is to provide the documentation for a future reference.
+A new developer might join the project in the future. The description will help him understand
+the motivation behind the implementation of a specific feature.    
+
+A new rails project created with the [Infinum rails template](https://github.com/infinum/default_rails_template)
+includes a [pull request description template](https://help.github.com/en/articles/creating-a-pull-request-template-for-your-repository). 
+The template is defined in the `rails_project/.github/PULL_REQUEST_TEMPLATE.md` file.
+
+```
+Task: [#__TASK_NUMBER__](__ADD_URL_TO_PRODUCTIVE_TASK__)
```

Comment:
@tomazzlender @d4be4st @melcha @nikone does it make sense to add URL to Productive task? the structure of the URL might change over time (if you had those URLs on a project from 3 years ago, it would certainly have changed) and you might end up with a dead link.
Also, the task could be private (not a good habit, but possible) and whoever opens the link won't be able to access it.

Not as likely, but no less important - tasks can be deleted.

Reply:
I believe it is. If I look at the PR, why would I manually search for the task, if I can just click on the link?

Most of the links will work most of the time. Of course, except if Productive changes URL schema for tasks. In that case, I think it's good to make a redirect from old schema to a new one in Productive app.

In any case, the problem with broken links is a global problem of the world wide web, not just this use case. Every link eventually becomes broken, but we still use them all the time.

Reply:
I don't see a problem with links to tasks. It's faster to click on it than to search for it manually. If they become broken, we still have task numbers and can manually search for them.

Reply:
Ok, makes sense.